app/javascript/components/pages/__test__/TripShow.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import TripShow from "../TripShow";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("When TripShow renders", () => {
+  test("displays TripShow heading", () => {
+    const trip = shallow(<TripShow />);
```

Comment:
looks like there needs to be some mock data for this test and also some assertions on data users expect to see on this page